### PR TITLE
Removed myself

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

### CODE REVIEW

**Review by rhuss — APPROVED** [review]
looks not good to me 😭 

/lgtm
/approve

**Comment by nak3** [comment]
:sob:

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -16,7 +16,6 @@ aliases:
   - yanweiguo
   channel-wg-leads:
   - matzew
-  - slinkydeveloper
```
Comment (rhuss) [inline OWNERS_ALIASES:19]: @slinkydeveloper Please revert the changes in this file and the other top-level owners_aliases again, the file is indeed auto-generated.

**Comment by slinkydeveloper** [comment]
@rhuss reverted :) Now can you let me go? :laughing:

**Comment by rhuss** [comment]
> @rhuss reverted :) Now can you let me go? 😆

No :P 

/lgtm
/approve

**Comment by csantanapr** [comment]
Why @slinkydeveloper 🥲 ?

@rhuss as TOC member team I think you should be added to repo approvers

**Comment by csantanapr** [comment]
Oh I think your already approver @rhuss
